migrations/2025-06-04-103308_multi-community/up.sql

```diff
@@ -0,0 +1,85 @@
+CREATE TABLE multi_community (
+    id serial PRIMARY KEY,
+    creator_id int NOT NULL REFERENCES person ON UPDATE CASCADE ON DELETE CASCADE,
+    instance_id int NOT NULL REFERENCES instance ON UPDATE CASCADE ON DELETE CASCADE,
+    name varchar(255) NOT NULL,
+    title varchar(255),
+    description varchar(255),
+    local bool NOT NULL DEFAULT TRUE,
+    deleted bool NOT NULL DEFAULT FALSE,
+    ap_id text UNIQUE NOT NULL DEFAULT generate_unique_changeme (),
+    public_key text NOT NULL,
+    private_key text,
+    inbox_url text NOT NULL DEFAULT generate_unique_changeme (),
+    last_refreshed_at timestamptz NOT NULL DEFAULT now(),
+    following_url text NOT NULL DEFAULT generate_unique_changeme (),
+    published timestamptz NOT NULL DEFAULT now(),
+    updated timestamptz
+);
+
+CREATE TABLE multi_community_entry (
+    multi_community_id int NOT NULL REFERENCES multi_community ON UPDATE CASCADE ON DELETE CASCADE,
+    community_id int NOT NULL REFERENCES community ON UPDATE CASCADE ON DELETE CASCADE,
```

Comment:
`community_id` needs to be indexed for the `community_used_in_multiple` function.